python/.pylintrc

```diff
@@ -45,7 +45,9 @@ limit-inference-results=100
 
 # List of plugins (as comma separated values of python module names) to load,
 # usually to register additional checkers.
-load-plugins=
+load-plugins=pylintfileheader
+
+file-header=(#.*\n)*?# Copyright \(C\) 2020 Dremio\n#\n# Licensed under the Apache License, Version 2\.0 \(the \"License\"\);\n# you may not use this file except in compliance with the License\.\n# You may obtain a copy of the License at\n#\n# http://www\.apache\.org/licenses/LICENSE-2\.0\n#\n# Unless required by applicable law or agreed to in writing, software\n# distributed under the License is distributed on an \"AS IS\" BASIS,\n# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.\n# See the License for the specific language governing permissions and\n# limitations under the License\.
```

Comment:
is it intentional that this regex seems to allow multiple lines in front of the Copyright?
do we need to support anything else besides a single shebang line?

Reply:
There was a flake8 noqa comment in one of the files- moved it. Changed the regex to only allow a shebang.